### PR TITLE
feat/tup-636: Migrate c-card--images to core-styles

### PR DESCRIPTION
**Pull request by R-Tomas-Gonzalez**
## Overview

Migrate c-card--images to core-styles

## Related
- [TUP-636](https://jira.tacc.utexas.edu/browse/TUP-636)
- Migrate from [c-card--images](https://github.com/TACC/tup-ui/blob/eab873fd3c479d7220ec4d2e747c43e967deba3f/apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/for-core-styles/components/c-card.css#L49-L158)

## Changes

1. Remove card--images styles from c-card--from-tup-cms
2. Added in new component to c-card called c-card--images.hbs
3. Added in specific styles for c-card--images.hbs pattern in c-card.demo.images.css
4. Added in section styles for c-card--images

## Testing

1. Check http://localhost:3000/components/detail/c-card--images

## UI

| Before | After |
| --- | --- |
| Screenshot 2023-11-02 at 3 38 03 PM | Screenshot 2023-11-02 at 3 38 47 PM |
| Screenshot 2023-11-02 at 3 43 06 PM | Screenshot 2023-11-02 at 3 43 37 PM |
| Screenshot 2023-11-02 at 3 47 21 PM | Screenshot 2023-11-02 at 3 47 33 PM |
| Screenshot 2023-11-02 at 3 49 24 PM | Screenshot 2023-11-02 at 3 49 31 PM |

### CODE REVIEW

File: src/lib/_imports/components/c-card.css [hunk 1]
```diff
@@ -186,3 +186,112 @@
     line-height: 1.7;
     margin-bottom: 2rem; /* to mimic <p> */
 }
+
```
Comment (wesleyboar) [inline src/lib/_imports/components/c-card.css:189]:
```suggestion





```

**Review by wesleyboar — COMMENTED** [review]
I made commits: a fix to [an outdated import], and some undo-ing of accidental [whitespace changes (to reduce diff)].

The `src/lib/_imports/components/c-card/c-card--images.hbs` looks like a lot of code. I think `c-card` is built such that this demo can be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)-er.

I don't have bandwidth yet to give further direction. I can re-evaluate when I am more available, or re-review if you have a DRY-er solution.

**Comment by R-Tomas-Gonzalez** [comment]
**Note. | Warning!** There is a side-effect, that I'm not sure off-hand how to deal with in the pattern library:
Screenshot 2023-11-06 at 4 41 39 PM
Screenshot 2023-11-06 at 4 42 38 PM

--image-width (right below that style) also has an effect on the image padding

**Review by wesleyboar — COMMENTED** [review]
Demo looks good.

1. The [side-effect discovered](https://github.com/TACC/Core-Styles/pull/254#issuecomment-1796985593) is an expected caveat.[^1] It is permissible.
2. Many templates were used to achieve the demo. I will attempt, in a PR off of your branch, to use less templates.

[^1]: The caveat is that if space in card for image is wider than image, then image can not fill up that space. I would entertain a solution in a follow-up PR, but for now, let's just migrate.

**Review by wesleyboar — APPROVED** [review]
**Approved.**

My reduction of templates can come later.

**Pull request merged**